TimberWinR/Manager.cs

```diff
@@ -256,6 +256,15 @@ public void ProcessConfiguration(CancellationToken cancelToken, Configuration co
                         output.Connect(elistner);
                 }
 
+                foreach (var stdin in config.Generators)
```

Comment:
Not related to this line, but I was reading the Manager.cs class to get my bearings and noticed that on line 27, the Udps list is declared as a List<TcpInputListener>. Was that intentional?